app/assets/javascripts/views/graphs/country_government.js

```diff
@@ -33,12 +38,35 @@
       }.bind(this));
     },
 
-    _drawGraph: function() {
+    _formatNum: function(num) {
+      if (num > 1000 || num < -1000) {
+        var num = '$' + d3.format('.3s')(num);
+        num = num.replace("G", "B");
+        return num;
+      } else {
+        return d3.round(num, 2);
+      }
+    },
 
+    _drawText: function(data) {
+      if ( data[0].cost > 0 ) {
```

Comment:
I'd set here not null. Just in case we have something with 0 as that is a valid data.